src/packages/ratmama/ratmama_parser.py

```diff
@@ -167,39 +172,48 @@
         sep = '-'
 
     if not sep:
-        ctx.bot.board.append(Rescue(irc_nickname=ctx.user.nickname, client=ctx.user.nickname))
+        ctx.bot.board.append(Rescue(irc_nickname=ctx.user.nickname.split("[")[0],
+                                    client=ctx.user.nickname))
         index = ctx.bot.board.find_by_name(ctx.user.nickname)
         await ctx.reply(f"Case #{index} created for {ctx.user.nickname}, please set details")
         return
     system: str = None
-    code_red: bool = False
+    cr: bool = False
     platform: Platforms = None
     for part in message.split(sep):
         part = part.strip()
         if part.casefold() in ("pc",):
             platform = Platforms["PC"]
+            message = re.sub(part, "", message, flags=re.IGNORECASE)
```

Comment:
[Discussion] [Not Blocking] Is there a case in which this `re.sub` shouldn't be called? It appears to be called in every branch of this `if`. Would it not be better placed outside of the `if`s to remove the duplication?

Reply:
it is not called in the event none of the conditions are met.